dkg/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright © Aptos Foundation
+
+use aptos_config::network_id::{NetworkId, PeerNetworkId};
+use aptos_network::application::{
+    error::Error,
+    interface::{NetworkClient, NetworkClientInterface, NetworkServiceEvents},
+};
+use aptos_types::PeerId;
+use aptos_validator_transaction_pool as vtxn_pool;
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+use tokio::runtime::Runtime;
+
+#[allow(clippy::let_and_return)]
+pub fn start_dkg_runtime(
```

Comment:
nit: now that these have been registered with the network, we'll be able to send messages to these interfaces.
I assume you're targeting `1.10` for the release? If not, it might make sense to config gate these interfaces. Otherwise, peers will be able to send messages to them (even if your implementation isn't ready for it 😄).